redbeams/src/main/java/com/sequenceiq/redbeams/flow/stack/AbstractRedbeamsAction.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.redbeams.flow.stack;
+
+import com.sequenceiq.cloudbreak.common.event.Payload;
+import com.sequenceiq.flow.core.AbstractAction;
+import com.sequenceiq.flow.core.CommonContext;
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.core.FlowState;
+
+import javax.annotation.PostConstruct;
+
+public abstract class AbstractRedbeamsAction<S extends FlowState, E extends FlowEvent, C extends CommonContext, P extends Payload>
```

Comment:
Currently, this class adds nothing over AbstractAction. Is it meant to be a stub for now, to be filled in later?

Reply:
I asked this question of @gergopapi2. It is the convention, but I don't know why exactly, but he recommended sticking with it.

Reply:
The idea was: let's have a base class in redbeams, so that we do not depend on directly a class in a different package. If we need to change something later for all of the classes in redbeams, or just to see the hierarchy. Other services also do have a base class.